docs/api-ref/rest-api/index.md

```diff
@@ -16,6 +16,60 @@ Prefect Cloud and a locally hosted Prefect server each provide a REST API.
 - Interactive REST API documentation for a locally hosted open-source Prefect server is available at <a href="http://localhost:4200/docs" target="_blank">http://localhost:4200/docs</a> or the `/docs` endpoint of the [`PREFECT_API_URL`](/concepts/settings/#prefect_api_url) you have configured to access the server. You must have the server running with `prefect server start` to access the interactive documentation.
 - The REST API documentation for a locally hosted open-source Prefect server is also available in the [Prefect REST API Reference](/api-ref/rest-api-reference/).
 
+## Interacting with the REST API
+
+You have many options to interact with Prefect REST API
+- create an instance of [`PrefectClient`](/api-ref/prefect/client/orchestration/#prefect.client.orchestration.PrefectClient) 
+- use your favorite Python HTTP library such as [requests](https://requests.readthedocs.io/en/latest/) or [httpx](https://www.python-httpx.org/)
```

Comment:
We should capitalize these to match our style guide

Reply:
I read this as capitalize the first word of each bullet point, not capitalize `requests` 🤷‍♀️